src/web/src/components/SearchProvider.tsx

```diff
@@ -1,40 +1,65 @@
 import { createContext, ReactNode, useState, useEffect, FormEvent } from 'react';
 import { useRouter } from 'next/router';
 
-type FilterProp = {
-  filter: 'post' | 'author' | string;
-};
-
 export interface SearchContextInterface {
```

Comment:
I'm curious as to whether we should default the advanced search features here. Might need to look into this possibility more